dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -97,6 +97,15 @@ public static void requireFATRunner(String className) {
                                             "class level.");
     }
 
+    @Override
+    protected String testName(FrameworkMethod method) {
+        String testName = method.getName();
+        if (RepeatTestFilter.CURRENT_REPEAT_ACTION != null && !RepeatTestFilter.CURRENT_REPEAT_ACTION.equals("NO_MODIFICATION_ACTION")) {
```

Comment:
could re-order the comparison to avoid the null check:
`if(!"NO_MODIFICATION_ACTION".equals(RepeatTestFilter.CURRENT_REPEAT_ACTION))`